exercises/atbash-cipher/Makefile

```diff
@@ -0,0 +1,17 @@
+solution := 
+
+chez := scheme
+guile := guile
+
+help :
+	echo "Test your solutions from the command line by calling make *, where * is the name of your scheme."
```

Comment:
I ti probably safer top use single quotes here.  I guess that the `make` probably doesn't do shell expansion first, but if somebody tries to just copy this line for some reason and run it in their terminal, then they will get all of the contents of their current directory everytime there is a `*` I think.

Reply:
haha good to know!